src/emulator/storage/apis/firebase.ts

```diff
@@ -128,6 +130,48 @@ export function createFirebaseEndpoints(emulator: StorageEmulator): Router {
     return res.json(new OutgoingFirebaseMetadata(metadata));
   });
 
+  firebaseStorageAPI.post(`/b/:bucketId/o/:objectId[(:)]generateSignedUrl`, async (req, res) => {
+    const timeToLive = req.body.ttl;
```

Comment:
Looks like it's going to be `req.body.ttlInMillis`, per [go/signedurl-js-proposal](http://goto.google.com/signedurl-js-proposal).